ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml

```diff
@@ -277,13 +391,55 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-disconnected
     workflow: cucushift-installer-rehearse-gcp-ipi-disconnected-private-cco-manual-workload-identity
+- as: gcp-ipi-disk-encryption-f28
+  cron: 12 19 11 * *
+  steps:
+    cluster_profile: gcp-qe
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-gcp-ipi-disk-encryption
+- as: gcp-ipi-marketplace-f28
+  cron: 3 8 21 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      COMPUTE_OSIMAGE: projects/redhat-marketplace-public/global/images/redhat-coreos-ocp-48-x86-64-202210040145
+      CONTROL_PLANE_OSIMAGE: projects/redhat-marketplace-public/global/images/redhat-coreos-ocp-48-x86-64-202210040145
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-gcp-ipi-workers-marketplace
+- as: gcp-ipi-mini-perm-custom-type-f28
+  cron: 58 14 3 * *
+  steps:
+    cluster_profile: gcp-qe
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-gcp-ipi-minimal-permission
+- as: gcp-ipi-oidc-rt-fips-regen-cert-f28
```

Comment:
The job is for api cert team, if you want to cover it, it should `gcp-ipi-oidc-rt-fips` and remove `chain: openshift-e2e-test-qe-regenerate-cluster-cert`

Reply:
This job exists in `openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml`:
```
- as: gcp-ipi-oidc-rt-fips-regen-cert-f28
  cron: 36 6 7 * *
  steps:
    cluster_profile: gcp-qe
    env:
      FIPS_ENABLED: "true"
      RT_ENABLED: "true"
    test:
    - chain: openshift-e2e-test-qe-regenerate-cluster-cert
    - chain: openshift-e2e-test-qe
    workflow: cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity
```

Reply:
@wangke19 Could you please let me know if the api cert e2e job need to be implemented in upgrade?